pkg/util/users/users.go

```diff
@@ -272,6 +293,16 @@ func quoteMemberList(user spec.PgUser) string {
 	return strings.Join(memberof, ",")
 }
 
+func revokeRole(groupRole, role string, db *sql.DB) error {
+	revokeStmt := fmt.Sprintf(revokeFromUserSQL, groupRole, role)
```

Comment:
a better name for `revokeFromUserSQL` would be `revokeGroupRoleFromRoleSQL`, for example the `cron_admin` is not a user in PG jargon because it is a non-login role.